project/api/models.py

```diff
@@ -33,8 +33,12 @@ class TerritorialEntity(models.Model):
     """
 
     wikidata_id = models.PositiveIntegerField()  # Excluding the Q
+    label = models.TextField(max_length=90)
     color = ColorField()
     admin_level = models.PositiveIntegerField()
+    inception_date = models.DecimalField(decimal_places=1, max_digits=10)
+    dissolution_date = models.DecimalField(decimal_places=1, max_digits=10)
```

Comment:
I assume that inception or dissolution dates can be unknown. For example, countries that are existing right now does not have a dissolution date.

Probably we should add `blank=True, null=True`